.scrutinizer.yml

```diff
@@ -0,0 +1,103 @@
+            constants: '(([A-Z_][A-Z0-9_]*)|(__.*__))$'
+            attributes: '[a-z_][a-z0-9_]{2,30}$'
+            arguments: '[a-z_][a-z0-9_]{2,30}$'
+            class_attributes: '([A-Za-z_][A-Za-z0-9_]{2,30}|(__.*__))$'
+            inline_vars: '[A-Za-z_][A-Za-z0-9_]*$'
+            classes: '[A-Z_][a-zA-Z0-9]+$'
+            modules: '(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$'
+            methods: '[a-z_][a-z0-9_]{2,30}$'
+        basic_missing_docstring: true
+        basic_missing_module_attribute: true
+        basic_nonexistent_operator: true
+        basic_not_in_loop: true
+        basic_old_raise_syntax: true
+        basic_return_outside_function: true
+        basic_unreachable: true
+        basic_useless_else_on_loop: true
+        classes_abstract_method: true
+        classes_access_member_before_definition: true
+        classes_bad_classmethod_argument: true
+        classes_bad_staticmethod_argument: true
+        classes_missing_interface_method: true
+        classes_no_self_argument: true
+        classes_no_self_use: true
+        classes_protected_access: true
+        design_abstract_class_little_used: true
+        design_abstract_class_not_used: true
+        exceptions_bare_except: true
+        exceptions_broad_except: true
+        exceptions_notimplemented_raised: true
+        exceptions_pointless_except: true
+        exceptions_raising_bad_type: true
+        exceptions_raising_non_exception: true
+        format_bad_indentation:
+            indentation: '4 spaces'
+        format_bad_whitespace: true
+        format_line_too_long:
+            max_length: '80'
+        format_lowercase_l_suffix: true
+        format_missing_final_newline: true
+        format_mixed_indentation: true
+        format_multiple_statements: true
+        format_old_ne_operator: true
+        format_superfluous_parens: true
+        format_trailing_whitespace: true
+        format_unnecessary_semicolon: true
+        imports_cyclic_import: true
+        imports_deprecated_module: true
+        imports_import_error: true
+        imports_import_self: true
+        imports_reimported: true
+        imports_relative_import: true
+        logging_format_truncated: true
+        newstyle_bad_super_call: true
+        open_mode_bad_open_mode: true
+        string_bad_format_string_key: true
+        string_constant_anomalous_backslash_in_string: true
+        string_mixed_format_string: true
+        string_too_few_format_args: true
+        string_too_many_format_args: true
+        typecheck_missing_kwoa: true
+        typecheck_not_callable: true
+        typecheck_redundant_keyword_arg: true
+        typecheck_too_many_function_args: true
+        typecheck_unexpected_keyword_arg: true
+        variables_global_at_module_level: true
+        variables_global_variable_not_assigned: true
+
+tools:
+    pylint:
+        python_version: '2'
+        config_file: pylintrc
+    external_code_coverage:
```

Comment:
CMIIW @ismailsunni :

I don't see .coveragerc files in InaSAFE, so I assume we don't have any external coverage service (like coveralls.io). So, we can either uses coverage service, or perhaps delete external_code_coverage node altogether since it causes errors.